.github/workflows/release.yaml

```
@@ -66,8 +66,19 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
     
-    - name: Create eks-charts PR
-      run: make ekscharts-sync-release
+    - name: Sync Helm Chart Catalog information
+      run: make sync-catalog-information-for-helm-chart
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_SESSION_TOKEN: ${{ secrets.AWS_SESSION_TOKEN }}
+    
+    - name: Sync Helm Chart to ECR Public
+      run: make push-helm-charts
```

```suggestion
      run: make push-helm-chart
```

Update to match make target name.